src/Silex/Translator.php

```diff
@@ -26,7 +26,17 @@ public function __construct(Application $app, $selector, $cacheDir = null, $debu
     {
         $this->app = $app;
 
-        parent::__construct(null, $selector, $cacheDir, $debug);
+        parent::__construct($this->app['locale'], $selector, $cacheDir, $debug);
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
Full inheritdoc as here must be without the {}

Reply:
This one is used throughout the whole Silex code, I don't want to change it.

Reply:
We can do that as follow up PR to cleanup
But that this the valid one for full overwrite and will help stan tooling here :)